README.md

```diff
@@ -21,13 +21,33 @@ Details are described later.
 - `src/config/configuration.ts`
 - `src/config/transactionAllowList.ts`
 
-### 3. Set up npm
+### 3. Set configuration variables
+- Create file `.env` refer to `.env.example`
+- Update those variables with corrected values from your side
+
+### 4. Set up redis server
+- Verse-Proxy use redis as a database to read and write data, so in order to run this proxy you will need start a redis server and pass url to `REDIS_URI`
+- also Verse-Proxy need to know which database you would use which can be config using `DATASTORE`
+For example: 
+```
+DATASTORE=redis
+REDIS_URI=redis://localhost:6373
+```
+
+### 5. Handle disconnect from node's websocket
+- `MAXRECONNECTATTEMPTS` specifies the maximum number of attempts the Verse-proxy will make to reconnect to a node's websocket.
+**example**: 
+```
+MAXRECONNECTATTEMPTS: 10
+```
+
+### 5. Set up npm
```

Comment:
@Sotatek-JohnnyNguyen 
There are two '5.'s.

> ### 5. Handle disconnect from node's websocket
> ### 5. Set up npm

Reply:
sorry, my bad. let me fix this